Liquibase does not compile on Mac (group id '612255346' is too big)

## Environment

**Liquibase Version**: 4.4.3/master

**Operating System Type & Version**: Mac OS (Catalina and up - probably just any version)

## Description

The _Liquibase_ project won't compile on a Mac, because the assembly plugin configuration needs to be adjusted. The error message is:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-assembly-plugin:3.1.1:single (make-assembly) on project liquibase-dist: Execution make-assembly of goal org.apache.maven.plugins:maven-assembly-plugin:3.1.1:single failed: group id '612255346' is too big ( > 2097151 ). Use STAR or POSIX extensions to overcome this limit -> [Help 1]
```

An easy fix would be to add `<tarLongFileMode>posix</tarLongFileMode>`to the plugin configuration:

```
           <plugin>
                <artifactId>maven-assembly-plugin</artifactId>
                <version>3.1.1</version>
                <configuration>
                    <tarLongFileMode>posix</tarLongFileMode>
                    <finalName>liquibase-${liquibase.version}</finalName>
                    <appendAssemblyId>false</appendAssemblyId>
                    <attach>false</attach>
                    <descriptors>
                        <descriptor>src/main/assembly/assembly-jar.xml</descriptor>
                        <descriptor>src/main/assembly/assembly-bin.xml</descriptor>
                    </descriptors>
                    <archive>
                        <manifestFile>${project.basedir}/../liquibase-core/target/classes/META-INF/MANIFEST.MF
                        </manifestFile>
                    </archive>
                </configuration>
```

--- a/liquibase-dist/pom.xml
+++ b/liquibase-dist/pom.xml
@@ -230,6 +230,7 @@
                 <artifactId>maven-assembly-plugin</artifactId>
                 <version>3.4.2</version>
 				<configuration>
+                    <tarLongFileMode>posix</tarLongFileMode>
                     <finalName>liquibase-${project.version}</finalName>
                     <appendAssemblyId>false</appendAssemblyId>
                     <attach>false</attach>

